src/ellipticLCS.jl

```diff
@@ -345,6 +344,10 @@ end
 """
     combine_31(singularities)
 
+Takes the list of singularities in `singularities` and combines them so that any -1/2 singularity whose three nearest
+neighbors are 1/2 singularities becomes an elliptic region, provided that the -1/2 singularity is in the triangle spanned
+by the wedges. This configuration is common for OECS, applying to material
+barriers on a large turbulet example yielded only about an additional 1% material barriers.
```

Comment:
Could you break the lines here earlier so that they fit into the usual editor window?